website/docs/add-secure-apps/providers/ssf/index.md

```diff
@@ -1,30 +1,39 @@
 
-Events in authentik that are tracked via SSF include when an MFA device is added or removed, logouts, sessions being revoked by Admin or user clicking logout, or credentials changed.
+### Transmitter and receiver
+
+The role of the transmitter is to send signals to the receiver, and the receiver is the application that listens for the signals. Each system receives and sends signals, but the signals are different.
+
+The identity provider is the transmitter of _identity signals_ from the IdP to the application,
+and the application is the receiver of _identity signals_, such as user information, groups, and roles.
+
+An SSF provider is the transmitter of _security signals_ from the IdP to the application,
+and the application is the receiver of _security signals_, such as device information, user events, and security events.
+
+This shared signals approach allows for more dynamic security decisions than traditional static policies, as access determinations can adapt based on both identity context and device posture in near real-time.
 
 ## Example use cases
 
 A common use case for SSF is when an Admin wants to know if a user logs out of authentik, so that the user is then also automaticlaly logged out of all other work-focused applications.
 
-Another example use case is when an application uses SSF to subscribe to authorization events because the application needs to know if a user changed their password in authentik. If a user did change their password, then the application receives a POST request to write the fact that the password was changed.
-
-## About using SSF in authentik
+Another example use case is when an application uses SSF to subscribe to authorization events because the application needs to know if a user changed their password in authentik. If a user changes their password, then the application will receive a POST request to relay an event that the password was changed.
 
-Let's look at a few details about using SSF in authentik.
+## SSF in authentik
 
-The SSF provider in authentik serves as a [backchannel provider](../../applications/manage_apps#backchannel-providers). Backchannel providers are used to augment the functionality of the main provider for an application. Thus you will still need to [create a typical application/provider pair](../../applications/manage_apps#instructions) (using an OIDC provider), and when creating the application, assign the SSF provider as a backchannel provider.
+SSF providers are configured in authentik by creating a [backchannel provider](../../applications/manage_apps#backchannel-providers). Backchannel providers are used to augment the functionality of the main provider for an application. Thus, you will still need to [create a typical application/provider pair](../../applications/manage_apps#instructions) (using an OIDC provider), and when creating the application, assign the SSF provider as a backchannel provider.
```

Comment:
I find his sentence confusing/too vague. The SSF provider *is* the backchannel provider. Assuming that is accurate, I want to be as clear as possible about that. Your rewording implies that one creates the SSF provider, and then also configure an additional entity, the backchannel provider.